### PR TITLE
[TS-203] BUG: Fixed position date and add checker

### PR DESCRIPTION
**Pull request by Ming424**
+ Fix date format for new added position
+ Add checker for input values: Symbol, Position Size, Valid Date (using snackbar for the notification)
image
image

### CODE REVIEW

File: src/components/Portfolio/Positions.vue [hunk 1]
```diff
@@ -246,12 +261,32 @@ export default {
...
+            this.symbolError = true; err = true;
+          }
+          if (this.editedStock.positionSize <= 0 && !isNaN(this.editedStock.positionSize)){
+            this.positionError = true; err = true;
+          }
+          if (!Date.parse(this.editedStock.date)) {
+            this.dateError = true; err = true;
+          }
+          if (err) {
+            this.snackbarText = "Unvalid input, please verify";
```
Comment (itsShaVawn) [inline src/components/Portfolio/Positions.vue:276]:
```suggestion
            this.snackbarText = "Invalid input. Please try again.";
```

Comment (itsShaVawn) [inline src/components/Portfolio/Positions.vue:276]:
```suggestion
            this.snackbarText = "Invalid input. Please try again.";
```

**Review by itsShaVawn — CHANGES_REQUESTED** [review]

File: src/components/Portfolio/Positions.vue (hunk 1 above)
Comment (Ming424) [reply src/components/Portfolio/Positions.vue]: Commited

**Review by itsShaVawn — DISMISSED** [review]
Lgtm 👍🏼

**Comment by Ming424** [comment]
BLOCK FTM INVESTIGATING THE ISSUE

**Comment by Ming424** [comment]
UNBLOCKED

**Review by kennguyen0303 — APPROVED** [review]
lgtm

**Comment by Ming424** [comment]
Also delete label component

**Review by tungleu — APPROVED** [review]
I prefer to have users use the calendar for date input instead of typing literal date format string which causes format inconsistency.  Overall the checker works as expected,good job!

**Pull request merged**